docs/index.md

```diff
@@ -1 +1,38 @@
-# Virtual Controlplane
\ No newline at end of file
+# Virtual Controlplane
+
+This project contains a collection of helm charts to deploy a virtual/nodeless [Kubernetes API server](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/).
```

Comment:
The project contains a collection of helm charts to deploy a virtual/nodeless [Kubernetes API server](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/).